src/ui/reducers/app.ts

```diff
@@ -48,6 +48,12 @@ const syncInitialAppState: AppState = {
 
 export async function getInitialAppState(): Promise<AppState> {
   const recordingId = getRecordingId();
+
+  // If we're in the library, there are no preferences to fetch.
+  if (!recordingId) {
```

Comment:
What do you think about wrapping the call below in an `if(recordingId)` rather than returning early `if(!recordingId)`? I could see someone adding some more initialization below this and getting very confused when it doesn't occur on non-recording pages.